src/redux/articles/reducer.ts

```diff
@@ -0,0 +1,199 @@
+import { createReducer, completeReducer, completeState } from 'redux-recompose';
+import Immutable from 'seamless-immutable';
+import { ArticlesState } from '@interfaces/reduxInterfaces';
+import { onPagination, onClearTarget } from '@utils/reduxUtils';
+
+import { actions } from './actions';
+
+const stateDescription = {
+  description: {
+    articlesList: {
```

Comment:
I'd move this list to the service

Reply:
I will remove the list when the integration is ready, the comment is already added with the TODO

Reply:
I understand but it should return a promise with the data in the service